addons/website_sale/models/website.py

```diff
@@ -772,12 +772,19 @@ def _get_and_cache_current_cart(self):
         SaleOrderSudo = self.env['sale.order'].sudo()
 
         sale_order_sudo = SaleOrderSudo
-        partner_sudo = self.env.user.partner_id
```

Comment:
What does it change here?
There will be only one user anyway, it shouldn't change anything to the prefetch and will avoid browsing the record multiple times no?